### PR TITLE
adding CITATION.cff (#136)

### LINKED ISSUE #136: Add CITATION.cff file
See example at [https://github.com/PCMDI/cmor/CITATION.cff](https://github.com/PCMDI/cmor/blob/82c1c4bac0526d3b835eb275da7c2fc044c05bd0/CITATION.cff)

@taylor13 @mauzey1 I will add this to my to-do in preparation for the CMOR 3.9.0 release

### PR DESCRIPTION
**Pull request by durack1**
Link https://github.com/PCMDI/cmor/issues/749
Fix #136 

The `2024-08-xx` will need to be updated when the 3.9.0 release of the PDF is released

@mauzey1 @taylor13 ping

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files
+# https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#definitionsperson
+
+cff-version: 1.2.0
+message: "If you use the CMOR3 software, please cite it using the metadata from this file"
+authors:
+- family-names: "Doutriaux"
+  given-names: "Charles"
+  orcid: "https://orcid.org/0009-0002-6932-3169"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
```
Comment (mauzey1) [inline CITATION.cff:10]:
```suggestion
- family-names: "Doutriaux"
  given-names: "Charles"
  orcid: "https://orcid.org/0009-0002-6932-3169"
  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI, formerly)"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  orcid: "https://orcid.org/0000-0003-1156-6774"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
+- family-names: "Nadeau"
+  given-names: "Denis"
+  orcid: "https://orcid.org/0000-0002-9622-3289"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
+- family-names: "Durack"
+  given-names: "Paul J."
+  orcid: "https://orcid.org/0000-0003-2835-1438"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
```
Comment (mauzey1) [inline CITATION.cff:26]:
```suggestion
- family-names: "Durack"
  given-names: "Paul J."
  orcid: "https://orcid.org/0000-0003-2835-1438"
  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI, formerly)"
```

**Review by mauzey1 — CHANGES_REQUESTED** [review]

File: CITATION.cff (hunk 2 above)
Comment (mauzey1) [reply CITATION.cff:26]: wrong section

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  orcid: "https://orcid.org/0000-0002-6491-2135"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
+- family-names: "Mauzey"
+  given-names: "Chris"
+  orcid: "https://orcid.org/0000-0003-1156-6774"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
+- family-names: "Nadeau"
+  given-names: "Denis"
+  orcid: "https://orcid.org/0000-0002-9622-3289"
+  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI)"
```
Comment (mauzey1) [inline CITATION.cff:22]:
```suggestion
- family-names: "Nadeau"
  given-names: "Denis"
  orcid: "https://orcid.org/0000-0002-9622-3289"
  affiliation: "Program for Climate Model Diagnosis and Intercomparison (PCMDI, formerly)"
```

File: CITATION.cff (hunk 1 above)
Comment (durack1) [reply CITATION.cff]: Added

**Review by mauzey1 — APPROVED** [review]

**Pull request merged**